pkgs/development/python-modules/blist/default.nix

```
@@ -14,6 +15,15 @@ buildPythonPackage rec {
     sha256 = "1hqz9pqbwx0czvq9bjdqjqh5bwfksva1is0anfazig81n18c84is";
   };
 
+
+  patches = [
+    # python 3.7 compatibility
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/DanielStutzbach/blist/pull/78.diff";
```

please dont use the pull request url, because it's not stable; for instance they can force push to it, hence changing the hash
```suggestion
      url = "https://github.com/DanielStutzbach/blist/commit/2dc1ec28ed68611fcec9ac1c68070c782d6b4b4e.patch";
```
also, can you add a link to PR in the comment? to keep a reference whether the patch was merged

---

seconded

---

resolved